### PR TITLE
Recreate the issue with the controller picking

### PR DESCRIPTION
**Pull request by rahul27**
GearVRf-DCO-1.0-Signed-off-by: Rahul Rudradevan
r.rudradevan@samsung.com

### CODE REVIEW

**Comment by NolaDonato** [comment]
I believe PR #1124 from GearVRF fixes this problem. But you need to use the new constructor which takes a GVRSceneObject instead of GVRContext:

mPicker = new GVRPicker(sphere, scene);

File: framework-tests/app/src/androidTest/java/org/gearvrf/tester/PickerTests.java [hunk 1]
```diff
@@ -690,4 +693,49 @@ public void pickQuadFromLeftAndRight()
         mPickHandler.checkNoHits("quad1");
         mPickHandler.checkNoHits("quad2");
     }
+
```
Comment (liaxim) [inline framework-tests/app/src/androidTest/java/org/gearvrf/tester/PickerTests.java]: Needs @Test

**Comment by liaxim** [comment]
Please update to use https://github.com/gearvrf/GearVRf-Tests/pull/78 so we can verify.

I am still curious where the fake events are coming from.

File: framework-tests/app/src/androidTest/java/org/gearvrf/tester/PickerTests.java (hunk 1 above)
Comment (rahul27) [inline framework-tests/app/src/androidTest/java/org/gearvrf/tester/PickerTests.java]: Done

Comment (NolaDonato) [inline framework-tests/app/src/androidTest/java/org/gearvrf/tester/PickerTests.java]: When picking from an object, the pick ray should be (0,0,0) to (0,0,1). I have made the picker default to this in the new constructor and now the demo works. This is pull request #1145

**Pull request merged**